src/test/beast/IPEndpoint_test.cpp

```diff
@@ -116,7 +116,6 @@ class IPEndpoint_test : public unit_test::suite
       BEAST_EXPECT(v4[2]==0);
       BEAST_EXPECT(v4[3]==1);
 
-      BEAST_EXPECT((!((0xff)<<16)) == 0x00000000);
```

Comment:
what do lines 119 and 120 have to do with this test (AddressV4) at all? I feel like I'm missing something.